test/e2e/e2e_suite_test.go

```diff
@@ -0,0 +1,61 @@
+package e2e
+
+import (
+	"os"
+	"testing"
+
+	ginkgo "github.com/onsi/ginkgo"
+	gomega "github.com/onsi/gomega"
+
+	"k8s.io/client-go/dynamic"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/tools/clientcmd"
+
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
+	"sigs.k8s.io/controller-runtime/pkg/log/zap"
+
+	clusterv1client "github.com/open-cluster-management/api/client/cluster/clientset/versioned"
+)
+
+func TestE2E(t *testing.T) {
+	gomega.RegisterFailHandler(ginkgo.Fail)
+	ginkgo.RunSpecs(t, "E2E suite")
+}
+
+var (
```

Comment:
minor: we've found that creating a helper to produce a per instance client allow better tracking in traces as each can different agent strings